Dockerfile

```diff
@@ -7,4 +7,4 @@ COPY . .
 RUN npm install
 RUN npm run bundle
 
-ENTRYPOINT ["node", "bin/har-to-k6.js"]
+ENTRYPOINT ["node", "bin/har-to-k6.js", "--stdout"]
```

Comment:
Docker defaults to `stdout`. If `-o, --output` is used, `--stdout` is overridden.